dandischema/consts.py

```diff
@@ -5,6 +5,7 @@
 # migrate has a guard now for this since it cannot migrate to anything but current
 # version
 ALLOWED_TARGET_SCHEMAS = [DANDI_SCHEMA_VERSION]
+ALLOWED_VALID_SCHEMAS = ALLOWED_TARGET_SCHEMAS + ["0.4.4"]
```

Comment:
what does it mean to be a "VALID" schema?  I thought they are all "valid" at the moment of their existence in the history.  Needs some kind of a comment here at the least since separation between TARGET and VALID is not clear.

Reply:
changed name and added comment.